pkg/config/policies.go

```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
 	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName], policyName, policies)
 }
 
+// AddChannelPolicy modifies an existing channel policy configuration.
+// When the policy exists it will overwrite the existing policy.
```

Comment:
Is this the ideal behavior for Add? Not saying it's wrong but we should consider whether we should allow the user to overwrite existing policies via the Add method instead of a Replace method or via a Delete followed by an Add

Reply:
make sense, thought about the same thing during the implementation. 
However, currently all add methods in policies.go share the same implementation, it's probably worth it to consider the ideal behavior, and apply it over the entire package.

Reply:
Add != Set; Set could imply Add but not the other way round, IMO.